test/persistence/failpoints/before.td

```diff
@@ -27,45 +28,53 @@ $ set schema={
+> CREATE SINK failpoint_sink FROM failpoint
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'testdrive-failpoint-sink-${testdrive.seed}'
+  KEY (f1)
+  WITH (reuse_topic=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+
+$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=10000
 {"f1": "c${kafka-ingest.iteration}"} {"f2": "c${kafka-ingest.iteration}"}
 
-$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=100000
+$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=10000
 {"f1": "d${kafka-ingest.iteration}"} {"f2": "d${kafka-ingest.iteration}"}
 
 # Make sure that we read (and persisted) at least one message before activating the failpoint.
 > SELECT COUNT(*) > 0 FROM failpoint;
 true
 
-> SET failpoints = 'fileblob_set_sync=return';
+> SET failpoints = '${arg.failpoint}=${arg.action}';
 
-$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=100000
+$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=10000
 {"f1": "e${kafka-ingest.iteration}"} {"f2": "e${kafka-ingest.iteration}"}
 
-$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=100000
+$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=10000
 {"f1": "f${kafka-ingest.iteration}"} {"f2": "f${kafka-ingest.iteration}"}
 
-$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=100000
+$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=10000
 {"f1": "g${kafka-ingest.iteration}"} {"f2": "g${kafka-ingest.iteration}"}
 
-$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=100000
+$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=10000
 {"f1": "h${kafka-ingest.iteration}"} {"f2": "h${kafka-ingest.iteration}"}
 
-$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=100000
+$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=10000
 {"f1": "i${kafka-ingest.iteration}"} {"f2": "i${kafka-ingest.iteration}"}
 
-$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=100000
+$ kafka-ingest format=avro topic=failpoint key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=10000
```

Comment:
is there something that we can query at the end of this test to verify that the source is in an errored state?

Reply:
I do not think we can do more at the end of this .td file that would work across all failpoints. The `panic` failpoints will crash Mz, so it can not be queried afterwards. For the `return` failpoints, which ones would be expected to put the source in an error state?